utils/past_ci_versions.py

```diff
@@ -0,0 +1,116 @@
+import argparse
```

Comment:
This file is to make the installation commands easier to read in Docker files.
This should be used for other Docker files once this PR is merged.

For example, the following block hurts my head a bit already 😄 

https://github.com/huggingface/transformers/blob/49cd736a288a315d741e5c337790effa4c9fa689/docker/transformers-pytorch-gpu/Dockerfile#L19-L21